Objects/EventItem/EventItemSpawner.gd

```diff
@@ -0,0 +1,27 @@
+extends Node
+
+var time_to_spawn_item: float = 5
+var temp_time: float = 0
+var SackOfGold = load("res://Objects/EventItem/SackOfGold.tscn")
+var DwarfInBalloon = load("res://Objects/EventItem/DwarfInBalloon.tscn")
+
+func set_next_item_time():
+	var value = randi()%60+20
+	time_to_spawn_item = value 
+	temp_time = 0
+
+func spawn_item():
+	var index: int = randi()%2+1 
+	if index == 1:
+		var sof = SackOfGold.instance()
+		get_parent().call_deferred("add_child", sof)
+	else:
+		var din = DwarfInBalloon.instance()
+		get_parent().call_deferred("add_child", din)
+
+func _process(delta):
+	temp_time += delta
```

Comment:
I think, it will be better when we use build-in Timer node instead of timer variable. Less code, more flexible and readable.